cartodb/cartodb.py

```diff
@@ -138,7 +143,7 @@ def get_response_data(self, resp, parse_json=True):
                 return resp.json()
             return resp.content
         elif resp.status_code == requests.codes.bad_request:
-            raise CartoDBException(resp.json()['error'])
+            raise CartoDBException(resp.json()['errors'])
```

Comment:
Sorry I somehow missed the notification for this. Could you simply try one and if KeyError is raised then go for the other option?